src/main/java/net/sf/jabref/model/DuplicateCheck.java

```diff
@@ -16,25 +18,28 @@
 import net.sf.jabref.model.entry.FieldProperty;
 import net.sf.jabref.model.entry.InternalBibtexFields;
 
+import info.debatty.java.stringsimilarity.Levenshtein;
```

Comment:
Since JabRef 3.6 is close to be available in Debian/unstable, there is no need for a library migration.